doc/source/ray-air/examples/batch_tuning.ipynb

```diff
@@ -0,0 +1,1154 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6f5c2fbd-31d7-4262-94c8-1f6cea6f5938",
+   "metadata": {},
+   "source": [
+    "# Running N Independent Trials in Parallel with Ray AIR Tune"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cdb02a92-a603-48b8-a23e-887ecaaf5470",
+   "metadata": {},
+   "source": [
+    "# Introduction \n",
+    "\n",
+    "Let us quickly walk through the key concepts you need to know to use Tune.\n",
+    "\n",
+    "First, you define the hyperparameters you want to tune in a `Search space` and pass them into a `Trainable` or `callable` function, that specifies the objective you want to tune.  The trainable function will be called for every instance in the search space.\n",
+    "\n",
+    "Then you select a search algorithm and optionally use a\n",
```

Comment:
We would want to stress that we are using Tune for a non-HPO related job. Something like "While Tune's main purpose is hyperparameter optimization, you can also use it as an execution engine to manage multiple independent workers, which is what we will do in this example"